tests/test_configurations.py

```diff
@@ -11,6 +13,36 @@ class SMRFTestCase(unittest.TestCase):
     the base config. Also will remove the output directory upon tear down.
     """
 
+    def compare_netcdf_files(self, gold_file, test_file, atol=1e-5):
+        """
+        Compare two netcdf files to ensure that they are identical. The
+        tests will compare the attributes of each variable and ensure that
+        the values are within a 1e-8 tolerance (from `np.allclose`)
```

Comment:
Are you wanting to use 1e-8 as the default for the kwarg atol which is currently 1e-5?

Reply:
This has been removed as it's now an exact test.